src/main/java/seedu/address/logic/parser/FindApptCommandParser.java

```diff
@@ -35,13 +40,13 @@ public FindApptCommand parse(String args) throws ParseException {
         }
 
         ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_NRIC,
-                        PREFIX_DATE, PREFIX_START_TIME, PREFIX_END_TIME);
+                ArgumentTokenizer.tokenize(args);
 
         argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_NRIC,
                 PREFIX_DATE, PREFIX_START_TIME, PREFIX_END_TIME);
```

Comment:
Maybe can remove PREFIX_NAME and PREFIX_END_TIME for duplicate checking here

Reply:
Will check the other files as well!